flaml/ml.py

```diff
@@ -438,11 +438,28 @@ def evaluate_model_CV(
     task,
     eval_metric,
     best_val_loss,
+    cv_score_agg_func,
     log_training_metric=False,
     fit_kwargs={},
 ):
+    if cv_score_agg_func is None:
+        def cv_score_agg_func(metrics_across_folds):
+            metric_to_minimize = sum([tem[0] for tem in metrics_across_folds])/len(metrics_across_folds)
+            metrics_to_log = None
+            for single_fold in metrics_across_folds:
+                if single_fold[1] is None:
+                    break
+                elif metrics_to_log is None:
+                    metrics_to_log = single_fold[1]
+                else:
+                    metrics_to_log = {k: metrics_to_log[k] + v for k, v in single_fold[1].items()}
+            if metrics_to_log:
+                n = len(metrics_across_folds)
+                metrics_to_log = {k: v / n for k, v in metrics_to_log.items()}
+            return metric_to_minimize, metrics_to_log

```

Comment:
Defining here will slow it down every time the function `evaluate_model_CV()` is called. Move the definition out.

Reply:
Maybe move out cv_score_agg_func (before evaluate_model_CV) and rename it default_cv_score_agg_func?